### PR TITLE
1.8.1 notes: sort PR

### PR DESCRIPTION
**Pull request by miekg**
Sort the PR list and add 2 TODOs for PR that needs to be merged before
we do a 1.8.1 release.

Signed-off-by: Miek Gieben <miek@miek.nl>

### CODE REVIEW

**Review by yongtang — APPROVED** [review]
/lgtm

File: notes/coredns-1.8.1.md [hunk 1]
```diff
@@ -15,25 +15,29 @@ The CoreDNS team has released
...
-* plugin/kubernetes: Fix NPE issue (https://github.com/coredns/coredns/pull/4338)
+* plugin/forward: perform HC every 0.5 (TODO)
+* plugin/health: Fix health check endpoint (https://github.com/coredns/coredns/pull/4231)
 * plugin/kubernetes: Add support for dual stack ClusterIP Services (https://github.com/coredns/coredns/pull/4339)
-* plugin/azure: Iterate over all RecordSetListResultPage Pages (https://github.com/coredns/coredns/pull/4351)
+* plugin/kubernetes: Fix dns programming duration metric (https://github.com/coredns/coredns/pull/4255)
+* plugin/kubernetes: Fix NPE issue (https://github.com/coredns/coredns/pull/4338)
+* plugin/kubernetes: Watch EndpointSlices (https://github.com/coredns/coredns/pull/4209)
+* plugin/local: add local plugin (https://github.com/coredns/coredns/pull/4262)
+* plugin/local: add local plugin (https://github.com/coredns/coredns/pull/4262)
```
Comment (mrueg) [inline notes/coredns-1.8.1.md:42]: @miekg this seems to be a duplicate of the previous line

**Comment by miekg** [comment]
[ Quoting <notifications@github.com> in "Re: [coredns/coredns] 1.8.1 notes: ..." ]
>@mrueg commented on this pull request.
>
>━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
>
>In notes/coredns-1.8.1.md:
>
>>  * plugin/kubernetes: Add support for dual stack ClusterIP Services (https://github.com/coredns/coredns/pull/4339)
>-* plugin/azure: Iterate over all RecordSetListResultPage Pages (https://github.com/coredns/coredns/pull/4351)
>+* plugin/kubernetes: Fix dns programming duration metric (https://github.com/coredns/coredns/pull/4255)
>+* plugin/kubernetes: Fix NPE issue (https://github.com/coredns/coredns/pull/4338)
>+* plugin/kubernetes: Watch EndpointSlices (https://github.com/coredns/coredns/pull/4209)
>+* plugin/local: add local plugin (https://github.com/coredns/coredns/pull/4262)
>+* plugin/local: add local plugin (https://github.com/coredns/coredns/pull/4262)
>
>@miekg this seems to be a duplicate of the previous line

ta. Need to do another pass anyway, will keep this in mind and delete the dup

**Pull request merged**